### PR TITLE
Fixed TinkerPop links

### PR DESCRIPTION
**Pull request by robertdale**

### CODE REVIEW

File: docs/hadoop.txt [hunk 1]
```diff
@@ -119,4 +119,4 @@ def defineGratefulDeadSchema(janusGraph) {
 Running PageRank
 ~~~~~~~~~~~~~~~~
 
-A fully functional example of the http://tinkerpop.incubator.apache.org/docs/3.0.1-incubating/#pagerankvertexprogram[PageRankVertexProgram] can be found in the http://tinkerpop.incubator.apache.org/docs/3.0.1-incubating/#vertexprogram[VertexProgram] section of the TinkerPop3 docs.
+A fully functional example of the http://tinkerpop.apache.org/docs/3.1.1-incubating/reference#pagerankvertexprogram[PageRankVertexProgram] can be found in the http://tinkerpop.apache.org/docs/3.1.1-incubating/reference#vertexprogram[VertexProgram] section of the TinkerPop3 docs.
```
Comment (pluradj) [inline docs/hadoop.txt]: should use the templated versions here, like `$MAVEN{tinkerpop.version}`

Comment (hsaputra) [inline docs/hadoop.txt]: Ah good catch

**Comment by hsaputra** [comment]
Will merge once tests passed

**Comment by mbrukman** [comment]
Thanks, @robertdale!

**Pull request merged**